src/dbt_score/__init__.py

```diff
@@ -1,11 +1,12 @@
 """Init dbt_score package."""
 
 from dbt_score.model_filter import ModelFilter, model_filter
```

Comment:
We have to adapt `dbt_score.model_filter` in a similar way as `dbt_score.rule` was changed in order to apply filters to sources as well

Reply:
Updated in fe80bdf!  I will update the argument name references here as well in another commit.

How would you like to address the naming of the class/decorator now that it applies to both `Model`s and `Source`s?  Is it okay to have a breaking change here to rename it `RuleFilter` or `EvaluableFilter` or something else?

Reply:
Nice! :raised_hands: I will have a look at the changes. 

I am afraid that this indeed warrants a breaking change, the name does not make sense anymore. But, breaking changes are expected considering we are still at version `0.6`. I like `RuleFilter` the most!